<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>837</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>682</width>
    <height>837</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>837</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="sum_deposit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 2px solid gray;
	font: 28pt &quot;.AppleSystemUIFont&quot;;
 	border-radius: 10px;
	background-color: white;</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_place">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1825</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="place_period">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;
border: 2px solid gray;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Срок размещения в днях</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="percen">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 2px solid gray;
	font: 28pt &quot;.AppleSystemUIFont&quot;;
 	border-radius: 10px;
	background-color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_sum_trans">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sum_trans">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 24pt &quot;.AppleSystemUIFont&quot;;
border: 2px solid gray;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cap_percen">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>  Капитализация
  процентов</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="in_transaction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="out_transaction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Снятия</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tax">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 2px solid gray;
	font: 28pt &quot;.AppleSystemUIFont&quot;;
 	border-radius: 10px;
	background-color: white;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждый год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>770</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid green;
 	border-radius: 25px;
	background-color:rgb(216, 214, 218);
	margin: 0px
}

QPushButton:pressed {
	background-color: green;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Сalculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="out_sum">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 24pt &quot;.AppleSystemUIFont&quot;;
border: 2px solid gray;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="out_percen">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>620</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 24pt &quot;.AppleSystemUIFont&quot;;
border: 2px solid gray;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="out_tax">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>690</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
font: 24pt &quot;.AppleSystemUIFont&quot;;
border: 2px solid gray;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе в конце срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="period_trans">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 2px solid gray;
	font: 28pt &quot;.AppleSystemUIFont&quot;;
 	border-radius: 10px;
	background-color: white;</string>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>770</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(255, 40, 21);
 	border-radius: 25px;
	background-color:rgb(216, 214, 218);
	margin: 0px
}

QPushButton:pressed {
	background-color: rgb(255, 40, 21);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_sum_trans</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_sum_trans</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sum_trans</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>sum_trans</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>place_period</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>place_period</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>place_period</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>place_period</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
